platform/commonUI/general/res/sass/search/_search.scss

```diff
@@ -227,7 +227,8 @@
 
 		.load-icon {
 			position: relative;
-			&.loading {
+
+/*			&.loading {
```

Comment:
Instead of commenting out, can we just remove this?  Git history will allow it to be recovered if we need it later.